fiblary3/common/restapi.py

```diff
@@ -89,7 +89,7 @@ def __init__(
             session = requests.Session()
         self.session = session
         self.session.verify = verify
-        self.session.user_agent = user_agent
+        self.session.headers['User-Agent'] = user_agent
```

Comment:
This appears to be the right way to set the header in recent versions of `requests`.